src/app/pre-sale/components/popup/index.tsx

```diff
@@ -26,21 +25,24 @@ export const PreSalePopup = (props: PreSalePopupProps) => {
     >
       <ModalOverlay className='bg-ic-black bg-opacity-60 backdrop-blur' />
       <ModalContent className='bg-transparent shadow-none'>
-        <ModalBody className='bg-transparent'>
-          <DepositProvider preSaleToken={getDefaultIndex()}>
+        <ModalBody className='dark bg-transparent'>
```

Comment:
dark mode to have components supporting dark mode update accordingly e.g. input component

Reply:
🕶️